src/components/style/tippy/popover-default-theme.css

```diff
@@ -0,0 +1,15 @@
+.tippy-box[data-theme~="popover-default"] {
+  color: initial;
+  background-color: var(--color-white);
+  border-radius: var(--border-radius-default);
+  box-shadow: 0 4px 5px 0 rgba(0, 0, 0, 0.14),
+    0 1px 10px 0 rgba(0, 0, 0, 0.12),
+    0 2px 4px -1px rgba(0, 0, 0, 0.2);
+  overflow: hidden;
```

Comment:
One thing to discuss: I sort of "don't like" using theme props to conditionally add classnames... I just think it's less flexible and a lot harder to grasp. Because there are more than 1 theme in tooltip, I opted to use classes, but I opted to use css for Popover.

Now given that, I opted to inline the box-shadow of the `elevation--2` class directly. One of my 1:1 items is that I don't really see a reason we can't encapsulate these into variables?

Reply:
We discussed offline and decided to skip for now?